system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -864,36 +864,58 @@
-                          :update-field "SCIENCE_KEYWORDS"
-                          ;; Note: find-value is case-sensitive.
-                          :find-value {:Topic "aTmoSPHERE"}
-                          :update-value {:Category "EARTH SCIENCE"
-                                         :Topic "ATMOSPHERE"
-                                         :Term "AIR QUALITY"
-                                         :VariableLevel1 "EMISSIONS"}}]
+        bulk-update-body1 {:concept-ids ["ALL"] 
+                           :update-type "FIND_AND_UPDATE"
+                           :update-field "SCIENCE_KEYWORDS"
+                           ;; Note: find-value is case-sensitive.
+                           :find-value {:Topic "aTmoSPHERE"}
+                           :update-value {:Category "EARTH SCIENCE"
+                                          :Topic "ATMOSPHERE"
+                                          :Term "AIR QUALITY"
+                                          :VariableLevel1 "EMISSIONS"}}
+        bulk-update-body2 {:concept-ids [(:concept-id coll1)] 
+                           :update-type "FIND_AND_UPDATE"
+                           :update-field "SCIENCE_KEYWORDS"
+                           ;; Note: find-value is case-sensitive.
+                           :find-value {:Topic "aTmoSPHERE"}
+                           :update-value {:Category "EARTH SCIENCE"
+                                          :Topic "ATMOSPHERE"
+                                          :Term "AIR QUALITY"
+                                          :VariableLevel1 "EMISSIONS"}}]
     ;; perform bulk update, verify that one collection is skipped because find-value is not found.
-    (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body)
+    (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body1)
           _ (index/wait-until-indexed)
           collection-response (ingest/bulk-update-task-status "PROV1" (:task-id response))]
       (is (= "COMPLETE" (:task-status collection-response)))
       (is (= "Task completed with 1 SKIPPED out of 1 total collection update(s)." (:status-message collection-response))))
+  
     ;; delete the collection
     (is (= 200 (:status (ingest/delete-concept (data2-core/umm-c-collection->concept coll1 :echo10)))))
     (index/wait-until-indexed)
-    
+   
     ;; perform another bulk update, verify that the deleted collection is not included when getting all 
     ;; collections from the provider. 
-    (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body)
+    (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body1)
           _ (index/wait-until-indexed)]  
       (is (= ["There are no un-deleted collections for provider-id [PROV1]."]
-             (:errors response))))))
-              
+             (:errors response))))
+
+    ;; perform a bulk update with the deleted collection's concept-id. It doesn't exclude tomb-stone one.
```

Comment:
Can we include a collection that isn't deleted with the list of concept-ids?

Reply:
Sure. Done.